src/Pages/AboutPage.js

```diff
@@ -0,0 +1,269 @@
+import Recliner from '../Assets/Images/recliner-icon.svg'
+import StationWagon from '../Assets/Images/station-wagon-icon.svg'
+import { Link } from "react-router-dom"
+
+function AboutPage() {
+  return (
+    <div>
+      <Navbar />
+
+      <div className="vision-began-container">
+        <div className="vb-header">
+            <SectionHeader 
+            text="How The Vision Began"
+            />
+        </div>
+
+        <div className="vb-text-content">
+          <p>Ever had one of those lightning-bolt-from-the-sky moments? I did. 
+          It arrived unannounced, typical lightning bolt! 
+          I was a hair stylist at the time with a solid clientele. I guess you could say I was on cruise control, stuck in limbo. Until I wasn’t...</p>
+          <br />
+          <p>I have always craved a challenge and I needed a change. 
+          So, I dove headfirst into an immersive full stack web development course. Adiós, dull routine, hello serious professional and creative freedom!
+          </p>
+          <br />
+          <p>
+          For the moment, hair styling still paid the bills, so I used that time  to hone into my logical side. and prepare myself for a new chapter. 
+          I styled hair during the day and designed websites for friends and family after hours. The excitement and satisfaction were intoxicating. 
+          </p>
+          <br />
+          <p>With all the logic under my belt now, it was time for me to devour design. I learned everything that I could get my little hands on about designing UI/UX to solve the worlds problems. Now, I am able  and craft digital masterpieces!. 
+          </p>
+          <br />
+          <p>And since I’ve got your ear--
+          Maybe we can change the world together.  Let’s collaborate. Or, check out my portfolio to see my work come to life. 
+          </p>
+        </div>
+
+        <div className="vb-image">
+          <img src={VBImage} alt="" />
+        </div>
+
+        <div className="vb-button">
+          <Link style={{textDecoration: 'none'}}to="#"><Button 
+          text="See My Resume"
+          /></Link> 
+        </div>
+
+        <div className="vb-section-break-line">
+        <SectionBreakLine />
+        </div>  
+      
+      </div>
+
+      <div className="skills-envisioned-container">
+        <div className="se-header">
+            <SectionHeader 
+            text="Skills Envisioned"
+            />
+        </div>
+
+        <div className="se-card-design">
+            <SECard 
+            header1="Design"
+            image={DesignCardImg}
+            alt="SVG of painters palette"
+            header2="Bring Your Audience Home"
+            para="I design beautiful sites with an organic connection to your brand. And a fiery dash of soul to keep your visitors on their toes. "
+            header3="Skills"
+            li1="UI / UX"
+            li2="Websites / Apps"
+            li3="Branding"
+            li4="Logos"
```

Comment:
the LI's could be an array, as mentioned above:
```suggestion
            lis={["UI / UX", "Websites / Apps", "Branding", "Logos"]}
```

Reply:
Hmm, thanks for the example. Not sure I fully understand it, but will research it so I can. Thanks again!